src/apps/dictionaries/components/DictionaryForm.tsx

```diff
@@ -215,34 +216,44 @@ const DictionaryForm: React.FC<Props> = ({
                 <ErrorMessage name="preferred_source" component="span" />
               </Typography>
             </FormControl>
-            <FormControl
-              fullWidth
-              // required
-              margin="normal"
-            >
-              <InputLabel htmlFor="owner_url">Owner</InputLabel>
-              <Field
-                value=""
-                disabled={editing || isSubmitting}
-                name="owner_url"
-                id="owner_url"
-                multiline
-                rowsMax={4}
-                component={Select}
-              >
-                {showUserName(profile)}
-                {showOrganisationHeader(usersOrgs)}
-                {showUserOrganisations(usersOrgs)}
-              </Field>
-              <Typography color="error" variant="caption" component="div">
-                <ErrorMessage name="owner_url" component="span" />
-              </Typography>
-            </FormControl>
-            <FormControl
-              fullWidth
-              // required
-              margin="normal"
-            >
+            {viewing || editing ? (
```

Comment:
Shouldn't we be using the same view for editing and creating and, if anything, have a different component for viewing?

Reply:
It is because we want this field be disabled in both view and edit mode that I employed this.

Reply:
@ibacher for this particular field of owner, we do not want to allow editing, correct me if I am wrong there, so that's why we are using a `TextField` for `viewing || editing` then creating will be a select dropdown.